### PR TITLE
[zh-cn]Update coarse-parallel-processing-work-queue.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview:
https://github.com/kubernetes/website/pull/35571/files?short_path=a51dfad#diff-a51dfada7551c3c8158e71f1710d9f7e6cad13167ed6a350791b8a2dd61e3acb

### CODE REVIEW

File: content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Next, we will check that we can discover the rabbitmq service:
 # Note the rabbitmq-service has a DNS name, provided by Kubernetes:
 -->
 ```
-# 请注意 rabbitmq-service 有Kubernetes 提供的 DNS 名称，
+# 请注意 rabbitmq-service 拥有一个有 Kubernetes 提供的 DNS 名称：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md:157]:
```suggestion
# 请注意 rabbitmq-service 拥有一个由 Kubernetes 提供的 DNS 名称：
```

File: content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md [hunk 2]
```diff
@@ -268,7 +266,7 @@ For our example, we will create the queue and fill it using the amqp command lin
 In practice, you might write a program to fill the queue using an amqp client library.
 -->
 
-本例中，如果有大量的数据需要被 Job 的所有 Pod 读取，典型的做法是把它们放在一个共享文件系统中，如NFS，并以只读的方式挂载到所有 Pod，或者 Pod 中的程序从类似 HDFS 的集群文件系统中读取。
+本例中，如果有大量的数据需要被 Job 的所有 Pod 读取，典型的做法是把它们放在一个共享文件系统中，如 NFS（Network File System 网络文件系统），并以只读的方式挂载到所有 Pod，或者 Pod 中的程序从类似 HDFS（Hadoop Distributed File System 分布式文件系统）的集群文件系统中读取。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue.md:269]:
```suggestion
本例中，如果有大量的数据需要被 Job 的所有 Pod 读取，典型的做法是把它们放在一个共享文件系统中，
如 NFS（Network File System，网络文件系统），并以只读的方式挂载到所有 Pod，或者 Pod 中的程序从类似
HDFS（Hadoop Distributed File System，分布式文件系统）的集群文件系统中读取。
```

**Review by windsonsea — CHANGES_REQUESTED** [review]
It's recommended to give some newline symbols to a long para.

**Review by Sea-n — COMMENTED** [review]
Overall looks good to me, wait for fixing previous suggestion brought by windsonsea.

Wow, I see you help updated the English comment even if the translated text is updated. 😊

---

The GitHub preview for Markdown is not the "preview page", we use Netlify bot to generate a live website for each PR.

Please refer to [this comment](https://github.com/kubernetes/website/pull/35571#issuecomment-1200119765), using "😎 Deploy Preview" with the right path:
https://deploy-preview-35571--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue

**Comment by yanrongshi** [comment]
> Overall looks good to me, wait for fixing previous suggestion brought by windsonsea.
> 
> Wow, I see you help updated the English comment even if the translated text is updated. 😊
> 
> The GitHub preview for Markdown is not the "preview page", we use Netlify bot to generate a live website for each PR.
> 
> Please refer to [this comment](https://github.com/kubernetes/website/pull/35571#issuecomment-1200119765), using "😎 Deploy Preview" with the right path: https://deploy-preview-35571--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/job/coarse-parallel-processing-work-queue

Thanks，my brother

**Pull request merged**